tests/pe-trial.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+require_relative "helpers/perf_run_helper"
+
+test_name "pe trial"
+
+teardown do
+  perf_teardown
+end
+
+# Setup
+## Create variables
+atop_result, gatling_result = nil
+
+## Set scenario file for test
+scenario_file = "pe-trial.json"
+
+## Get scenario data from scenario config file
+scenario_dir = File.expand_path("../simulation-runner/config/scenarios", __dir__)
+scenario = JSON.parse(File.read(File.join(scenario_dir, scenario_file)))
+
+## Get sim name and hit count from simulation config file identified in scenario data
+sim_dir = File.expand_path("../simulation-runner/src/main/scala/com/puppetlabs/gatling/node_simulations", __dir__)
+sim = File.basename(scenario["nodes"][0]["node_config"], ".json")
+sim_file = File.join(sim_dir, "#{sim}.scala")
+sim_hits = File.read(sim_file).scan(/exec\(http/).count
+
+## Set assertions
+## This is where changes should be made
+max_avg_mem = 3_000_000
+min_success_request_perc = 100
+max_response_time = 20_000 # magic
+total_request_count = scenario["nodes"][0]["num_instances"] * scenario["nodes"][0]["num_repetitions"] * sim_hits
+gatlingassertions = %W[SUCCESSFUL_REQUESTS=#{min_success_request_perc}
+                       MAX_RESPONSE_TIME_AGENT=#{max_response_time}
+                       TOTAL_REQUEST_COUNT=#{total_request_count}].join " "
+## End setup
+
+# Execute agent runs to warm up the JIT before starting our monitoring.
```

Comment:
Don't need a warm up for this test.  It should be able to pass cold.

Reply:
:+1: removed!